### ISSUE TITLE
It cannot be used on Windows now

### ISSUE THREAD

**Issue by wAnFen1017** [0x6a69616e/kpsdk-solver#18]

**Comment by pydlv** [comment]
What service are you using it against? What is your environment? E.g. bare metal windows w/ residential IP?

**Comment by wAnFen1017** [comment]
Sorry for my reckless response. I am in a Windows environment, and my own Google browser can make normal requests, but it cannot be successful in Playwright's Firefox. It has been tested multiple times

**Comment by pydlv** [comment]
It also recently stopped working for me against twitch.tv on Windows. Are you using a different service? If so, it's possible Kasada updated and broke this library.

**Comment by wAnFen1017** [comment]
That's right, I'm looking for a solution to crack it. If I find it, I will share it here

**Comment by 0x6a69616e** [comment]
https://github.com/0x6a69616e/kpsdk-solver/issues/21#issuecomment-2868276804
